### PR TITLE
Constant name is now default str representation unless explicit repre…

### PR DESCRIPTION
**Pull request by jMyles**
…sentation is set.

### CODE REVIEW

**Review by KPrasch — APPROVED** [review]
This is absolutely bonkers.

**Review by michwill — APPROVED** [review]

**Comment by jMyles** [comment]
Shoutout to @dmpayton for adding some sanity to my thinking about this.

File: constant_sorrow/tests.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import pytest
+
 from constant_sorrow import constants
 
 
 def test_establishing_a_constant():
     # You get a constant by just picking an all-caps name and importing it.
-    from constant_sorrow.constants import THIS_IS_A_VALID_CONSTANT
```
Comment (cygnusv) [inline constant_sorrow/tests.py:7]: Why removing this line? It's the single most important line on this file.

Comment (jMyles) [reply constant_sorrow/tests.py:7]: Riiight.  That was a mistake.

**Pull request merged**